Audacia.Quest/Audacia.Quest.Core/Renderer/SpriteRenderer.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Numerics;
+
+namespace Audacia.Quest.Core.Renderer
+{
+    public class SpriteRenderer
+    {
+        public SpriteRenderer(string imageSource)
+        {
+            ImageSource = imageSource;
+        }
+
+        public string ImageSource { get; set; }
```

Comment:
More of a personal preference really - I like to have all properties above the constructor. Not that arsed for this project but just something to consider :)